android-release-build.md

```diff
@@ -161,8 +164,9 @@ Create `/android/app/proguard-rules.pro` file and add rules listed below.
 -keep class io.flutter.plugins.**  { *; }
 ```
 
-The configuration above will only protect Flutter libraries.
-Any additional libraries e.g. Firebase will require their own rules to be added.
+The configuration above only protects Flutter libraries.
```

Comment:
I'd specify 'Flutter engine libraries' to avoid confusion

Reply:
Done. Thanks, @xster